projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -265,6 +265,10 @@ export interface OccEndpoints {
    * @member {string}
    */
   getActiveCostCenters?: string | OccEndpoint;
+  /**
+   * Get cart validation results
+   */
+  validate?: string | OccEndpoint;
```

Comment:
this one should go to cart-lib, right?

Reply:
You're right.  I pushed a commit to improve this.